README.md

```diff
@@ -18,7 +18,7 @@ This repository lists resources and projects, highlighting official and communit
 - [About MDN](https://developer.mozilla.org/en-US/about) - MDN's mission, team, partners, and projects.
 - [Curriculum](https://mdn.dev/curriculum) - Structured learning resources and modules for web development education.
 - [Content Repo](https://github.com/mdn/content) - Source repository for MDN's (en-US) documentation and examples.
-- [Platform Repo](https://github.com/mdn/yari) - Repository for the MDN platform, tools and site infrastructure.
+- [Platform Repo](https://github.com/mdn/yari) - Repository for the MDN platform, tools, and site infrastructure.
```

Comment:
With the switchover to rari in progress, it's basically Yari = frontend, rari = backend.

```suggestion
- [Yari](https://github.com/mdn/yari) - MDN's React-based frontend, including tools for building the site.
```